doc/source/advanced.md

```diff
@@ -279,3 +279,21 @@ prePuller:
   continuous:
     enabled: true
 ```
+
+### Pre-pulling additional images
+
+By default, the pre-puller only pulls the singleuser image & the networktools image (if
+access to cloud metadata is disabled). If you have customizations that need additional
+images present on all nodes, you can ask the pre-puller to also pull an arbitrary number
+of additional images.
+
+```yaml
+prePuller:
+   extraImages:
+     ubuntu:
```

Comment:
can each entry just after `extraImages` be named anything? It's a bit hard to tell since you're calling it `ubuntu` but it's also just pulling `ubuntu`

Reply:
Yes, it can be named anything (no underscores I think). Do you think you'll have time to edit this and push changes too? :)

Reply:
yeah I can take an editing pass before we merge, on this point I just wasn't sure what the actual behavior was. We don't yet have docs on actually using these extra images, right?

Reply:
Yeah, we don't have any docs. That can come later I think - my hope is to have @ryanlovett figure out all the best practices here and then we can document them.